membership-attribute-service/app/actions/CommonActions.scala

```diff
@@ -37,7 +37,7 @@ class CommonActions(touchpointBackends: TouchpointBackends, bodyParser: BodyPars
 class BackendRequest[A](val touchpoint: TouchpointComponents, request: Request[A]) extends WrappedRequest[A](request)
 
 class AuthenticatedUserAndBackendRequest[A](
-    val user: Either[AuthenticationFailure, UserFromToken],
```

Comment:
Is this only used down below where it used to be converted directly to Option?

Reply:
Well, it had to be handled all over the place.
But it's actually unnecessary - if request is set (right), this one is set (right) as well.

Reply:
So this is an actual refactoring - no functionality is changed or code paths lost (except unused ones).